api/src/main/java/com/flipkart/poseidon/internal/ParamValidationFilter.java

```diff
@@ -116,6 +118,16 @@ public void filterResponse(Request request, Response response) throws InternalEr
                         } catch (IOException e) {
                             logger.error("Error in reading body : {}", e.getMessage());
                         }
+                    } else if (bodyBytes != null && bodyBytes.length > 0) {
```

Comment:
Can we update UT for this new body param as bytes?

Reply:
done